tests/test_renderers.py

```diff
@@ -4,12 +4,37 @@
                         unicode_literals)
 
 import unittest
+import platform
+
+import pytest
 
 from gdal2mbtiles.renderers import JpegRenderer, PngRenderer, TouchRenderer
 from gdal2mbtiles.gd_types import rgba
 from gdal2mbtiles.utils import intmd5
 from gdal2mbtiles.vips import VImageAdapter
 
+# https://bugs.python.org/issue1322
+if platform.system() == 'Linux':
+    import distro
+    DISTRIBUTION = distro.linux_distribution()
+else:
+    DISTRIBUTION = platform.system_alias(
+        platform.system(),
+        platform.release(),
+        platform.version()
+    )
+
+PNG8_OS_HASHES = {
+    ('Ubuntu', '16.04', 'xenial'): 106831624867432276165545554861383631224,
+    ('Ubuntu', '18.04', 'bionic'): 93909651943814796643456367818041361877,
+    ('Ubuntu', '20.04', 'focal'): 226470660062402177473163372260043882022,
```

Comment:
Hm, how annoying the hash isn't stable. This will require more maintenance down the line.

Reply:
Very, but I'm putting it down to potential improvements in their renderer between versions, though I haven't measured file size ;)